docs/source/getting_started/overview.rst

```diff
@@ -29,6 +29,9 @@ PDF, Vega, Vega-Lite, etc.) to be opened as a file or returned by a
 kernel as rich output. See :ref:`file-and-output-formats` for more
 information.
 
+To navigate the user interface, JupyterLab offers :ref:`customizable keyboard shortcuts <shortcuts>`
+and the ability to use key-maps from vim, emacs, and Sublime.
```

Comment:
I think we should say the full name: `Sublime Text`

Reply:
As an aside, that's how we list it in the UI I believe.  Perhaps we should change that?

Reply:
Hm, I think it would be good to always called it Sublime Text.